tests/testSourceTable.py

```diff
@@ -219,17 +228,17 @@ def testForwarding(self):
 
     def testBitsColumn(self):
         allBits = self.catalog.getBits()
-        someBits = self.catalog.getBits(["a.flags", "c.flags"])
-        self.assertEqual(allBits.getMask("a.flags"), 0x1)
-        self.assertEqual(allBits.getMask("b.flags"), 0x2)
-        self.assertEqual(allBits.getMask("c.flags"), 0x4)
+        someBits = self.catalog.getBits(["a_flag", "c_flag"])
+        self.assertEqual(allBits.getMask("a_flag"), 0x1)
+        self.assertEqual(allBits.getMask("b_flag"), 0x2)
+        self.assertEqual(allBits.getMask("c_flag"), 0x4)
         self.assertEqual(someBits.getMask(self.fluxFlagKey), 0x1)
         self.assertEqual(someBits.getMask(self.shapeFlagKey), 0x2)
-        self.assert_(((allBits.array & 0x1 != 0) == self.catalog.columns["a.flags"]).all())
-        self.assert_(((allBits.array & 0x2 != 0) == self.catalog.columns["b.flags"]).all())
-        self.assert_(((allBits.array & 0x4 != 0) == self.catalog.columns["c.flags"]).all())
-        self.assert_(((someBits.array & 0x1 != 0) == self.catalog.columns["a.flags"]).all())
-        self.assert_(((someBits.array & 0x2 != 0) == self.catalog.columns["c.flags"]).all())
+        self.assert_(((allBits.array & 0x1 != 0) == self.catalog.columns["a_flag"]).all())
+        self.assert_(((allBits.array & 0x2 != 0) == self.catalog.columns["b_flag"]).all())
+        self.assert_(((allBits.array & 0x4 != 0) == self.catalog.columns["c_flag"]).all())
+        self.assert_(((someBits.array & 0x1 != 0) == self.catalog.columns["a_flag"]).all())
+        self.assert_(((someBits.array & 0x2 != 0) == self.catalog.columns["c_flag"]).all())
 
     def testCast(self):
         baseCat = self.catalog.cast(lsst.afw.table.BaseCatalog)
```

Comment:
Is the schema.setVersion(1) in testSlotUndefine() of this test suite still necessary?